cpu/mips_pic32mz/Makefile.include

```diff
@@ -4,15 +4,15 @@ include $(RIOTCPU)/mips_pic32_common/Makefile.include
 include $(RIOTMAKE)/arch/mips.inc.mk
 
 # define build specific options
-export CFLAGS += -march=m5101 -mmicromips -DSKIP_COPY_TO_RAM
-export CFLAGS += -DMIPS_MICROMIPS
+CFLAGS += -march=m5101 -mmicromips -DSKIP_COPY_TO_RAM
+CFLAGS += -DMIPS_MICROMIPS
 
 export LINKFLAGS += -Wl,--defsym,__use_excpt_boot=0 $(CFLAGS)
 export LINKFLAGS += -Tpic32mz2048_uhi.ld
 
 # the pickit programmer (MPLAB-IPE) wants physical addresses in the hex file!!
-export OBJCOPY = objcopy #use system objcopy as toolchain one is broken.
-export OFLAGS += \
+OBJCOPY = objcopy #use system objcopy as toolchain one is broken.
```

Comment:
this also exported in `makefiles/toolchain/{llvm,gnu}.inc.mk`, don't know if it makes sense to change that here too

Reply:
As I said in the PR description "This currently does not un-export them from makefiles/toolchain."

The reasoning is that I prefer changing the `toolchain` in its specific PR.
It is not supposed to be different, but as it would change almost each line, I think it deserve a smaller PR to ease review.